code/game/objects/items/weapons/storage/book.dm

```diff
@@ -42,7 +42,8 @@
 			if(affecting.status == ORGAN_ORGANIC) //No Bible can heal a robotic arm!
 				if(affecting.heal_damage(heal_amt, heal_amt, 0))
 					H.update_damage_overlays(0)
-	return
+					return 1
```

Comment:
This will stop the for loop right after one limb is healed instead of going through all limbs.